cpp/include/cugraph/graph_view.hpp

```diff
@@ -285,6 +289,20 @@ class graph_base_t {
   bool is_symmetric() const { return properties_.is_symmetric; }
   bool is_multigraph() const { return properties_.is_multigraph; }
 
+  void attach_edge_mask(edge_property_view_t<edge_t, uint32_t const*, bool> edge_mask_view)
+  {
+    edge_mask_view_ = edge_mask_view;
+  }
+
+  void clear_edge_mask() { edge_mask_view_ = std::nullopt; }
```

Comment:
I have no preference.  Is there a reason to prefer this syntax over `edge_mask_view_.reset()`?

Reply:
But consistency matters, so if we are using `.reset()` elsewhere in our codebase, I am OK to make a change.

Reply:
That was my assessment.  A quick check in the code, I see a few calls to `.reset()` and a bunch more assignments to `std::nullopt`.

I'm going to merge this PR.  We should probably pick a way and create a PR to update things to be consistent.